### DISCUSSION TITLE
[Windows] installation failing to find libmagic

### DISCUSSION THREAD

**Discussion by ble2877** [VoxelCubes/PanelCleaner#71] [Q&A] [answered]
I'm trying to install the code and run it but this message keeps popping up(line 49, in load_lib
raise ImportError('failed to find libmagic. check your installation')
ImportError: fail to find libmagic. check your installation)

**Comment by VoxelCubes** [comment]
https://github.com/VoxelCubes/PanelCleaner/blob/master/docs/faq.md

**Comment by VoxelCubes** [comment, marked as answer]
I've added a little guard in the dependency list now to default to installing the python-magic-bin on Windows, hopefully that will help starting with the next release.

For now, the FAQ should help you out.

**Reply by ble2877** [reply]
![image_2024-02-26_081009268](https://github.com/VoxelCubes/PanelCleaner/assets/158649703/2f2aa349-3c35-44a7-86fd-190c2399d814)
thank you so much it is working now

**Comment by VoxelCubes** [comment]
Awesome!
